tubular/scripts/retire_one_learner.py

```diff
@@ -125,7 +125,7 @@ def _get_learner_and_state_index_or_exit(config, username):
         learner = config['LMS'].get_learner_retirement_state(username)
         learner_state_index = _get_learner_state_index_or_exit(learner, config)
         return learner, learner_state_index
-    except HttpNotFoundError:
+    except HTTPError:
```

Comment:
ditto
maybe you should check for the exact 404 status code in the places like this? and maybe add the separate log instead